generators/app/templates/.gitignore

```diff
@@ -1,3 +1,35 @@
+################################################
+# macOS
```

Comment:
My preference would be to aggregate our git-ignores from our last 10 or so projects, remove anything that is specific to a project/CMS, and use that, since that would probably most accurately represent what we actually _need_ in our gitignore with as little fluff as possible.